test/runtime/test_format_ignores.py

```diff
@@ -0,0 +1,220 @@
+import errno
+import sys
+import tempfile
+from collections.abc import Generator
+from pathlib import Path
+
+import pytest
+
+tool_path = str(Path(__file__).parent.parent.parent)
+if tool_path not in sys.path:
+    sys.path.insert(0, tool_path)
+
+from tool.format_ignores import (  # noqa: E402
+    _process_directory,  # noqa: PLC2701
+    _process_file,  # noqa: PLC2701
+    _sort_ignore_list,  # noqa: PLC2701
```

Comment:
it's fine by me if you want to add `PLC2701` to the `.ruff.toml`, or add a `ruff: noqa: PLC2701` at the top of this file